### PR TITLE
Feature/admin groups

### PR DESCRIPTION
**Pull request by Yarovji**

### CODE REVIEW

File: src/app/admin-panel/admin-panel.module.ts [hunk 1]
```diff
@@ -7,36 +7,58 @@ import { MaterialModule } from '../material';
...
-import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { DialogEntryComponent } from './teachers/teachers-list/dialog/dialog-overview';
 import { StudentsListComponent } from './students-list/students-list.component';
-import { MatListModule } from '@angular/material/list';
 import { PlatformModule } from '@angular/cdk/platform';
-import { MatButtonModule } from '@angular/material/button';
-import { MatSelectModule } from '@angular/material/select';    
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { GroupsComponent } from './admin-panel/groups/groups.component';
+import {MatRadioModule} from '@angular/material/radio';
```
Comment (dmitryy90) [inline src/app/admin-panel/admin-panel.module.ts]: fix indentation

File: src/app/admin-panel/admin-panel/groups/add-modify/add-modify.component.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import { FormGroup } from '@angular/forms';
+import { MatBottomSheetRef, MAT_BOTTOM_SHEET_DATA } from '@angular/material';
+
+@Component({
+  selector: 'app-add-modify',
+  templateUrl: './add-modify.component.html',
+  styleUrls: ['./add-modify.component.scss']
+})
+
+export class AddModifyComponent implements OnInit {
```
Comment (dmitryy90) [inline src/app/admin-panel/admin-panel/groups/add-modify/add-modify.component.ts]: use meaningful name

File: src/app/services/groups.service.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      })
+    );
+  }
+
+  /**
+  * Method send changes in class or creates a new class
+  * @param group - data about the class that we want to change or create
+  * @returns - edit class - if "id" is true, or add new class  - if there is't "id"
+  */
+  addGrup(group: Group) {
```
Comment (dmitryy90) [inline src/app/services/groups.service.ts]: use typing

**Comment by Yarovji** [comment]
done

File: package.json [hunk 4]
```diff
@@ -23,6 +23,7 @@
     "@angular/platform-browser-dynamic": "~7.2.0",
     "@angular/router": "~7.2.0",
     "core-js": "^2.5.4",
+    "hammerjs": "^2.0.8",
```
Comment (dmitryy90) [inline package.json]: why do we need this?

**Pull request merged**